### PR TITLE
Chore - add PS module metadata to projects

### LINKED ISSUE #65: Provide package metadata
**Is your feature request related to a problem? Please describe.**
Provide package metadata to improve the consumer experience and make it easier to use.

**Describe the solution you'd like**
Add the following information to every `psd1`:
- Include `PowerShellVersion` (see [sample](https://github.com/Azure/azure-powershell/blob/master/src/ApiManagement/ApiManagement/Az.ApiManagement.psd1#L38))
- Add `Tags`, `LicenseUri`, `ProjectUri`, `IconUri` in private data (see [sample](https://github.com/Azure/azure-powershell/blob/master/src/ApiManagement/ApiManagement/Az.ApiManagement.psd1#L186))

### LINKED ISSUE #70: Provide link to release notes in psd1
**Is your feature request related to a problem? Please describe.**
Provide release notes for this first version.

**Describe the solution you'd like**
The release notes should contain an overview of the currently available modules/operations and can link to the module-specific documentation.
The PS-module can either contain the release notes in itself or contain a link to the actual release notes.

### PR DESCRIPTION
**Pull request by stijnmoreels**
Closes #65 
Closes #70

### CODE REVIEW

File: src/Arcus.Scripting.DataFactory/Arcus.Scripting.DataFactory.psd1 [hunk 1]
```diff
@@ -84,7 +84,39 @@ AliasesToExport = '*'
 # FileList = @()
 
 # Private data to pass to the module specified in RootModule/ModuleToProcess
-# PrivateData = ''
+PrivateData = @{
+
+    PSData = @{
+
+        # Tags applied to this module. These help with module discovery in online galleries.
+        Tags = 'Azure','Data Factory', 'Arcus'
```
Comment (tomkerkhove) [inline src/Arcus.Scripting.DataFactory/Arcus.Scripting.DataFactory.psd1:92]: Let's add `Data`
```suggestion
        Tags = 'Azure', 'Data', 'Data Factory', 'Arcus'

```

File: src/Arcus.Scripting.DataFactory/Arcus.Scripting.DataFactory.psd1 [hunk 2]
```diff
@@ -84,7 +84,39 @@ AliasesToExport = '*'
...
+        LicenseUri = 'https://github.com/arcus-azure/arcus.scripting/blob/master/LICENSE'
+
+        # A URL to the main website for this project.
+        ProjectUri = 'https://github.com/arcus-azure/arcus.scripting'
+
+        # A URL to an icon representing this module.
+        IconUri = 'https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png'
+
+        # ReleaseNotes of this module
+        ReleaseNotes = ''
```
Comment (tomkerkhove) [inline src/Arcus.Scripting.DataFactory/Arcus.Scripting.DataFactory.psd1:104]: Let's leave that out

File: src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psd1 [hunk 3]
```diff
@@ -84,7 +84,39 @@ AliasesToExport = '*'
 # FileList = @()
 
 # Private data to pass to the module specified in RootModule/ModuleToProcess
-# PrivateData = ''
+PrivateData = @{
+
+    PSData = @{
+
+        # Tags applied to this module. These help with module discovery in online galleries.
+        Tags = 'Azure','Key Vault', 'Arcus'
```
Comment (tomkerkhove) [inline src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psd1:92]: Let's add `Security` 
```suggestion
        Tags = 'Azure', 'Security', 'Key Vault', 'Arcus'

```

File: src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psd1 [hunk 4]
```diff
@@ -84,7 +84,39 @@ AliasesToExport = '*'
...
+        LicenseUri = 'https://github.com/arcus-azure/arcus.scripting/blob/master/LICENSE'
+
+        # A URL to the main website for this project.
+        ProjectUri = 'https://github.com/arcus-azure/arcus.scripting'
+
+        # A URL to an icon representing this module.
+        IconUri = 'https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png'
+
+        # ReleaseNotes of this module
+        ReleaseNotes = ''
```
Comment (tomkerkhove) [inline src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psd1:104]: Same here for all

File: src/Arcus.Scripting.Storage/Arcus.Scripting.Storage.psd1 [hunk 5]
```diff
@@ -84,7 +84,39 @@ AliasesToExport = '*'
 # FileList = @()
 
 # Private data to pass to the module specified in RootModule/ModuleToProcess
-# PrivateData = ''
+PrivateData = @{
+
+    PSData = @{
+
+        # Tags applied to this module. These help with module discovery in online galleries.
+        Tags = 'Azure','Table Storage', 'Arcus'
```
Comment (tomkerkhove) [inline src/Arcus.Scripting.Storage/Arcus.Scripting.Storage.psd1:92]:
```suggestion
        Tags = 'Azure', 'Storage', 'Data', 'Table Storage', 'Arcus'
```

File: src/Arcus.Scripting.DataFactory/Arcus.Scripting.DataFactory.psd1 (hunk 2 above)
Comment (stijnmoreels) [reply src/Arcus.Scripting.DataFactory/Arcus.Scripting.DataFactory.psd1]: Ok.

File: src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psd1 (hunk 4 above)
Comment (stijnmoreels) [reply src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psd1]: Yes.

**Comment by tomkerkhove** [comment]
A dynamic link to the release itself or the overall release page?

**Comment by mbraekman** [comment]
> A dynamic link to the release itself or the overall release page?

If we can link directly to the information of that specific release, that would be great!
If that isn't possible, to overall release page would do.

**Comment by stijnmoreels** [comment]
> > A dynamic link to the release itself or the overall release page?
> 
> If we can link directly to the information of that specific release, that would be great!
> If that isn't possible, to overall release page would do.

Ok, see #70 for this. We will be able to link.

**Pull request merged**